unittests/restart_chain_tests.cpp

```diff
@@ -232,7 +235,8 @@ BOOST_FIXTURE_TEST_CASE(test_light_validation_restart_from_block_log_with_pruned
    BOOST_REQUIRE_NO_THROW(block_log::repair_log(blocks_dir));
 }
 
-BOOST_AUTO_TEST_CASE(test_trim_blocklog_front) {
+
+void trim_blocklog_front() {
```

Comment:
I think this should pass in an optional<uint32_t> initial_default_version, optional<uint32_t> pre_trim_default_version, uint32_t expected_trim_version and call set_version before creating the initial block log and then before calling trim_block_log_front and then verify the version for the trimmed block log.

Reply:
this is already verified by calling smoke_test, so resolving this